core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -330,6 +330,13 @@ public String toString() {
           .setIgnoredSiteProperty(true)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
+  public static final PropertyKey USER_LOGS_DIR =
+      new Builder(Name.USER_LOGS_DIR)
+          .setDefaultValue(String.format("${%s}/user", Name.LOGS_DIR))
+          .setDescription("The path to store user log files.")
```

Comment:
nit: I think someone reading this without background might not know what "user log files" are. Could we say something similar to the tune of "logs from alluxio client operations"? Even I'm not a huge fan of my suggestion, but I think I just want to make it more clear it's logs from client operations.

Reply:
+1 for clarifying what a user log contains. How about "logs from alluxio CLI operations"?

Reply:
Fixed.